### ISSUE TITLE
Notes per link

### ISSUE THREAD

**Issue by Gandalf-the-Blue** [linkwarden/linkwarden#554]
**Is your feature request related to a problem? Please describe.**
I use Linkwarden for reading articles, and I'd like to have notes included in my Linkwarden so I can jot them down easily and have them in a central place.

**Describe the solution you'd like**
Ability to add notes for each link

**Describe alternatives you've considered**
Having notes in a separate app

**Additional context**
NAA

**Comment by arran4** [comment]
Agreed. Currently use AnyType for links, but this is one issue in terms of a complete migration.
